routes/api/report.js

```diff
@@ -88,6 +89,30 @@ module.exports = function(connection) {
             }
         });
     });
+
+    router.get('/report/countries', function (req, res, next) {
```

Comment:
Shouldn't this be `/api/report/countries`?

Reply:
That is being handled [here](https://github.com/openmrs/openmrs-contrib-atlas/blob/master/routes/api/index.js#L6).